### DISCUSSION TITLE
Extend Slot Recipes

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [cschroeter/park-ui#97] [Q&A] [answered]
Is there a way to extend a slot recipe (like a dialog for example) in the theme?

**Comment by cschroeter** [comment]
Like adding another part / slot?

**Reply by juliomuhlbauer** [reply]
No, more like in styling.
Like, I want to style the color of the backdrop on all dialogs. It would be cool to do that in the theme file.

**Reply by cschroeter** [reply, marked as answer]
@juliomuhlbauer 
Just as described in the [docs](https://park-ui.com/docs/panda/theme/customize)

```jsx
  theme: {
    extend: {
      slotRecipes: {
        dialog: {
          base: {
            backdrop: {
              background: 'purple',
            },
          },
        },
      }
```
